### PR TITLE
Implement CCCChange Checker

### LINKED ISSUE #274: Check CCCChanges
Calculating validator rewards in Dynamic Validator is complicated. We need to check that the calculation results are correct.

We can check the correctness by the method below.
Ask CodeChain for the balance between block n and block n-1 and compare it to the sum of the CCCChanges in block n. These two values must be the same.

### PR DESCRIPTION
**Pull request by majecty**
This PR depends on #279.
This PR closes #274

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -38,6 +39,10 @@ async function runServer() {
         );
     });
     context.worker.run();
+
+    if (process.env.ENABLE_CCC_CHANGES_CHECK) {
```
Comment (joojis) [inline src/index.ts]: Would you add it to README?

File: README.md [hunk 2]
```diff
@@ -79,6 +79,13 @@ yarn run start
 # You can change the host of CodeChain and DB host on the config/dev.json
 ```
 
+### Check CCCChanges
+
+If you run the Indexer with the environment `ENABLE_CCC_CHANGES_CHECK` variable,
+the Indexer checks to see if CCCChanges is well calculated.
+If you want to receive an email when an error is found, please set `SENDGRID_API_KEY` and `SENDGRID_TO` variables.
```
Comment (ScarletBlue) [inline README.md]: If you want to receive an email when an error is found, please set the `SENDGRID_API_KEY` and `SENDGRID_TO` variables.

File: README.md [hunk 3]
```diff
@@ -79,6 +79,13 @@ yarn run start
...
+If you want to receive a slack notification, please set `SLACK_WEBHOOK` variable.
```
Comment (ScarletBlue) [inline README.md]: If you want to receive a slack notification, please set the `SLACK_WEBHOOK` variable.

**Comment by majecty** [comment]
@ScarletBlue I fixed them.

**Pull request merged**